### PR TITLE
Add frontend features

### PR DESCRIPTION
**Pull request by ShubhamAtri**
I added frontend features to my portfolio.
These features includes :- 
1. Short introduction about myself
2. Carousel of my favourite animes/movies image
3. Links to external social networking and coding platforms.

[Link](https://satri-sps-summer20.el.r.appspot.com/) to view pesonal portfolio

### CODE REVIEW

**Comment by nityendrasingh** [comment]
Hi Shubham, Can you please add more text/description to this code review? I don't understand what this frontend does.

If you want more systematic approach then use https://github.com/ShubhamAtri/my-portfolio/issues

**Comment by ShubhamAtri** [comment]
@nityendrasingh Added more explanation in description of code review.

**Comment by nityendrasingh** [comment]
nice!

**Review by nityendrasingh — APPROVED** [review]

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -15,14 +15,47 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+// function addRandomGreeting() {
```
Comment (nityendrasingh) [inline portfolio/src/main/webapp/script.js:18]: Left over code?

File: portfolio/pom.xml [hunk 2]
```diff
@@ -35,7 +35,7 @@
         <version>2.2.0</version>
         <configuration>
           <!-- TODO: set project ID. -->
```
Comment (nityendrasingh) [inline portfolio/pom.xml:37]: leftover

File: portfolio/src/main/webapp/script.js (hunk 1 above)
Comment (ShubhamAtri) [reply portfolio/src/main/webapp/script.js:18]: There was no need for this function so I commented it initially. I forgot to delete it while pushing my code. I will delete this in next pull request.

**Pull request merged**